### PR TITLE
New Cobra Web binding

### PR DESCRIPTION
**Pull request by laves**

### CODE REVIEW

File: binding/web/src/cobra.ts [hunk 1]
```diff
@@ -286,19 +327,19 @@ export class Cobra implements CobraEngine {
     }
 
     return {
-      frameLength: frameLength,
-      inputBufferAddress: inputBufferAddress,
+      malloc: exports.malloc,
```
Comment (ksyeo1010) [inline binding/web/src/cobra.ts:330]: is there a reason we put malloc here?

Comment (laves) [reply binding/web/src/cobra.ts:330]: I was just copying Cheetah's exports - does Cheetah need it? What is it used for?

Comment (ksyeo1010) [reply binding/web/src/cobra.ts:330]: doesnt need it, I removed it

Comment (laves) [reply binding/web/src/cobra.ts:330]: cool I'll do the same

**Review by ksyeo1010 — APPROVED** [review]

**Comment by ksyeo1010** [comment]
lgtm, I guess you can add what we discussed yesterday in this PR or as a separate PR.

**Comment by laves** [comment]
I guess I'll add it to this one because I haven't released the package yet.

**Review by ksyeo1010 — APPROVED** [review]

**Pull request merged**